Gulpfile.mjs

```diff
@@ -222,33 +214,46 @@ function getFiles(glob, { include, exclude }) {
   return stream;
 }
 
-function buildBabel(exclude) {
-  const base = monorepoRoot;
+function createWorker(useWorker) {
+  const numWorkers = require("os").cpus().length / 2 - 1;
```

Comment:
I imagine we can/might want to make this configurable at some point?

Reply:
Yeah. Currently the default value is assuming the CPU has hyper-threading so it havles and minus the main thread. I tested different values locally, based on our scenario, increasing the `numWorkers` more than that does not yield faster performance.